src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -504,168 +491,150 @@ namespace CondenserLoopTowers {
 
             //   fluid bypass for single speed tower
             if (state.dataIPShortCut->lAlphaFieldBlanks(11) || AlphArray(11).empty()) {
-                state.dataCondenserLoopTowers->towers(TowerNum).CapacityControl = CapacityCtrl::FanCycling; // FanCycling
+                tower.CapacityControl = CapacityCtrl::FanCycling; // FanCycling
             } else {
-                {
-                    auto const SELECT_CASE_var(UtilityRoutines::MakeUPPERCase(AlphArray(11)));
-                    if (SELECT_CASE_var == "FANCYCLING") {
-                        state.dataCondenserLoopTowers->towers(TowerNum).CapacityControl = CapacityCtrl::FanCycling;
-                    } else if (SELECT_CASE_var == "FLUIDBYPASS") {
-                        state.dataCondenserLoopTowers->towers(TowerNum).CapacityControl = CapacityCtrl::FluidBypass;
-                    } else {
-                        state.dataCondenserLoopTowers->towers(TowerNum).CapacityControl = CapacityCtrl::FanCycling;
-                        ShowWarningError(state,
-                                         cCurrentModuleObject + ", \"" + state.dataCondenserLoopTowers->towers(TowerNum).Name +
-                                             "\" The Capacity Control is not specified correctly. The default Fan Cycling is used.");
-                    }
-                }
+                tower.CapacityControl =
+                    static_cast<CapacityCtrl>(getEnumerationValue(CapacityCtrlNamesUC, UtilityRoutines::MakeUPPERCase(AlphArray(11))));
+                if (tower.CapacityControl == CapacityCtrl::Invalid) {
```

Comment:
I'm not sure how this is actually going to work now with epJSON parser doing these checks and terminating the program on errors.